README.md

```diff
@@ -27,13 +27,13 @@ The [installation guide](https://github.com/onnx/turnkeyml/blob/main/docs/instal
 
 TurnkeyML is designed to support the following use cases. Of course, it is also quite flexible, so we are sure you will come up with some use cases of your own too.
 
-| Use Case               | Description |
```

Comment:
I like the new information, but each word we add at the top is an extra word that someone will NOT read at the bottom.

I don't think 5% of people will read anything after the long "Demo" section, for example.

If we want to add this new information, what about moving the "What's inside" section before "Use Cases"? It is a short section and we might want people to understand that before they try to understand "advanced" commands.

Reply:
Discussed offline; we will add some kind of GIF at the top of the readme to really grab people's attention. But for now we will merge this changeset as-is.